web/gen/backend_consts.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+import asyncio
+import json
+import textwrap
+from pathlib import Path
+
+from mitmproxy import version
+
+here = Path(__file__).parent.absolute()
+
+filename = here / "../src/js/backends/consts.ts"
+
+
+async def make() -> str:
+    data = {
+        "version": version.VERSION,
+    }
+
+    # language=TypeScript
+    content = (
+        "/** Auto-generated by web/gen/backend_consts.py */\n"
+        "export const BACKEND_CONSTS = %s\n"
```

Comment:
Why not just `export const VERSION = ...`? :)

Reply:
because we'll have also other constants. So, I was thinking we could access them like this: 
`BACKEND_CONSTS.version` or `BACKEND_CONSTS.protocols`